src/components/WhaleSightingViewer/WhaleSightingViewer.tsx

```diff
@@ -1,62 +1,62 @@
+import { useEffect, useState } from "react";
+import SightingList from "./SightingList";
+import { fetchAllApprovedSightings, WhaleSighting, getPendingSightings } from "../../clients/apiClient";
 import { Login } from "../Login/Login";
 import "./WhaleSightingViewer.scss";
 import { WhaleSightingFilters } from "./WhaleSightingFilters";
-import { useState } from "react";
 
 interface WhaleSightingViewerProps {
-	loggedIn: boolean;
+	loggedIn: boolean,
+	isAdminPage: boolean;
 }
 
-export function WhaleSightingViewer({ loggedIn }: WhaleSightingViewerProps) {
-  const [selectedWhaleSpecies, setSelectedWhaleSpecies] = useState("");
-	const [selectedColour, setSelectedColour] = useState("");
-	const [selectedTailType, setSelectedTailType] = useState("");
-	const [selectedSize, setSelectedSize] = useState("");
-	const [minLat, setMinLat] = useState("");
-	const [maxLat, setMaxLat] = useState("");
-	const [minLog, setMinLog] = useState("");
-	const [maxLog, setMaxLog] = useState("");
-
-	function handleSearch(event: any) {
-		event.preventDefault();
+export function WhaleSightingViewer({loggedIn, isAdminPage} : WhaleSightingViewerProps) {
+
+	const [page, setPage] = useState(1);
+
+	const [sightings, setSightings] = useState<WhaleSighting[]>();
+
+	if (!isAdminPage) {
+		useEffect(() => {
+			fetchAllApprovedSightings()
+				.then(data => setSightings(data));
+		}, [])
```

Comment:
This is good, could we have the if condition inside the useEffect instead?
```suggestion
	useEffect(() => {
		if (!isAdminPage) {
			fetchAllApprovedSightings()
				.then(data => setSightings(data));
		}
	        else {
	                getPendingSightings()
				.then(data => setSightings(data));
		}, [])
	}
```